src/main/java/graphql/nadel/NadelExecutionHints.java

```diff
@@ -0,0 +1,33 @@
+package graphql.nadel;
+
+public class NadelExecutionHints {
+    private final boolean optimizeExecution;
```

Comment:
I think `optimizeExecution` is too broad a name.

So really you have done a "optimise of there is no renames or hydration"

Later you might have some other sort of optimisation - so the name `optimizeExecution` covers that as well.  Meaning it too broad

Call it what it is - shortcutOnNoRenamesOrHydration say

Naming is hard - but this name is too broad

Reply:
I've changed it to `optimizeOnNoTransformations`, as there are 4 conditions to be simultaneously met and that would make for a very long name.

Naming is important so if others don't think "no transformations" is clear enough, I'm also happy to go with a longer name